Ginger/GingerCoreNET/ALMLib/ZephyrEnt/ZephyrEntCore.cs

```diff
@@ -160,6 +160,7 @@ public override bool ExportExecutionDetailsToALM(BusinessFlow bizFlow, ref strin
             {
                 return zephyrEntRepository.GetLoginProjects(ALMCore.DefaultAlmConfig.ALMUserName, ALMCore.DefaultAlmConfig.ALMPassword, ALMCore.DefaultAlmConfig.ALMServerURL);
             }).Result;
+            SetConnectValidationStatus(false);
```

Comment:
Need to set as False in case user changes any value related to the connection (user/pass/URL)

Reply:
removed